sql-plugin/src/main/scala/com/nvidia/spark/rapids/higherOrderFunctions.scala

```diff
@@ -402,8 +405,9 @@ case class GpuArrayExists(
   }
 
   override protected def transformListColumnView(
-    lambdaTransformedCV: cudf.ColumnView
-  ): GpuColumnVector = {
+                                                  lambdaTransformedCV: cudf.ColumnView,
+                                                  arg: cudf.ColumnView
```

Comment:
Same indentation question here, not consisten with ident used for `GpuArrayFilter` args.